test/unit/rules/no-duplicate-id-test.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+const generateRuleTests = require('../../helpers/rule-test-harness');
+
+const ERROR_MESSAGE = require('../../../lib/rules/no-duplicate-id').ERROR_MESSAGE;
+
+generateRuleTests({
+  name: 'no-duplicate-id',
+
+  config: true,
+
+  good: [
+    // Unique sibling TextNode IDs
+    '<div id="id-00"></div><div id="id-01"></div>',
+
+    // Ignore MustacheStatements, even duplicates
+    '<div id={{this.divId}}></div>',
+    '<div id={{this.divId}}></div><div id={{this.divId}}></div>',
```

Comment:
I guess technically we could detect this simple case, but I guess it's fine for now the way it currently works :)